src/com/projectkorra/projectkorra/waterbending/SurgeWave.java

```diff
@@ -299,7 +300,7 @@ public void progress() {
 							final Vector vec = GeneralMethods.getOrthogonalVector(this.targetDirection, angle, i);
 							final Block block = this.location.clone().add(vec).getBlock();
 
-							if (!blocks.contains(block) && (ElementalAbility.isAir(block.getType()) || block.getType() == Material.FIRE) || this.isWaterbendable(block)) {
+							if (!blocks.contains(block) && (ElementalAbility.isAir(block.getType()) || FireAbility.isFire(block.getType())) || this.isWaterbendable(block)) {
```

Comment:
There's no need for you to use FireAbility here, isFire is part of ElementalAbility so it can be used directly in the water abilities / air abilities

Reply:
Oop you right